src/containers/Review/overview/Overview.tsx

```diff
@@ -68,9 +87,57 @@ export const Overview: React.FC<{
                 <strong>{strings.REVIEW_OVERVIEW_SERIAL}: </strong>
                 {serial}
               </p>
+              {applicantDeadline && deadlineActive && (
+                <p className="right-item">
+                  <strong>{strings.REVIEW_OVERVIEW_DEADLINE}: </strong>
+                  {DateTime.fromJSDate(applicantDeadline).toLocaleString()}
+                </p>
+              )}
```

Comment:
The logic for whether the deadline shows up is slightly different to whether or not to display the "Extend" button/selector. For the deadline, we only show it if it's active (i.e. not expired) and exists. But for the "Extend" button, we also need to show it when it's *not* active (i.e. expired), but also the application needs to be either "Expired" or have status "Changes requested".